metadata.rb

```diff
@@ -0,0 +1,19 @@
+# Encoding: utf-8
+
+name             'cerner_splunk_ingredient'
+maintainer       'Cerner Innovation, Inc.'
+maintainer_email 'splunk@cerner.com'
+license          'Apache 2.0'
+description      'Installs Splunk'
+long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
+version          '1.0.0'
```

Comment:
Something to think about, do we want to start at 1.0.0 or do we want to start with 0.1.0 ?